msi/nunit/engine-files.wxi

```diff
@@ -10,7 +10,10 @@
             <Component Id="NUNIT.ENGINE.CORE" Location="local" Guid="F4F14D75-6096-467E-82E1-BD513645C0EF">
```

Comment:
Changes to the msi look good - but we need to sort out an addins file for the agents I think, or we'll break some existing extensions that are currently packaged with the msi. Same comments on the NuGet package also apply here, I think we'll need a separate addins file.

Reply:
Agreed!